client/style.css

```diff
@@ -0,0 +1,6 @@
+body {
+    font-family: 'Montserrat', sans-serif !important;
+    line-height: 1.6 !important;
+    color: #333 !important;
```

Comment:
Rather than having these colors here, add sass files to override bootstrap defaults.

Reply:
Good point